doc/syscalls/90001_screen.md

```diff
@@ -30,11 +30,24 @@ The screen driver allows the process to write data to a framebuffer of a screen.
 
     **Returns**: SUCCESS_U32 with 1 if yes, and 0 if no
 
+  * ### Command number: `2`
+
+    **Description**: Set power
+
+    **Argument 1**: 0 if off, nonzero if on.
+
+    **Argument 2**: unused
+
+    **Returns**: Ok(()) followed by the ready callback if the command was successful,
+    BUSY if another command is in progress,
+    INVAL if a brightness value is required but wasn't provided.
```

Comment:
I might vouch for chips to select a default brightness value rather than present a requirement that set_brightness be called before set_power, especially since the language here suggests that requirement might only be present for some screens

Reply:
Okay, this makes sense. I think it warrants some more changes in brightness: brightness changes are not guaranteed to take effect when the display is off.

Initially I intended the separation to make flashlight-like use cases possible, but a well-manufactured module will be a reasonable flashlight only when the pixels are set to white, and that needs the display to be on anyway.

Reply:
I think that not guaranteeing brightness changes when the display is off seems reasonable

Reply:
You updated the doc comments in `screen.rs`, but this still suggests that a brightness value may be required, can that line be removed?

Reply:
Thanks for catching that. Updated.